transport.validator.mapcss

```diff
@@ -120,6 +127,7 @@ relation.pt_route[!to]
 relation.pt_route["fixme:relation"="order members"]
 {
 	throwWarning: tr("The stops may not be in the right order");
+	-osmoseItemClassLevel: none;
```

Comment:
why not this one ?

Reply:
I should have one. But this commit only map rules to rep-existing  item/class in Osmose.